scripts/supervisord/assemble

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Do we need this script?
It looks like we are not using this script anywhere.

Reply:
OK, I can see that this is used in Dockerfile for supervisord image.
But still failing to understand why we would need to do anything with /opt/appr-root in it